src/rshim.c

```diff
@@ -2321,6 +2323,12 @@ static int rshim_handle_ownership_transfer(rshim_backend_t *bd)
       RSHIM_INFO("rshim%d regained ownership successfully\n", bd->index);
     }
   } else {
+    if (rshim_force_cmd_pending[bd->index]) {
+      RSHIM_INFO("Skipping Force cmd for rshim%d as it's already attached\n",
```

Comment:
"Skipping OwnershipReq cmd for rshim%d (already attached)"?

Reply:
I had to use"cmd" to keep line width under 80 :-(. Your version seems a good one that fits in 80 characters. Will try.